ci/build_cpp.sh

```diff
@@ -19,8 +19,13 @@ rapids-logger "Begin C++ and Python builds"
 
 sccache --zero-stats
 
+LIBRMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1739 cpp)
+RMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1739 python)
```

Comment:
Do we want to merge with those here or wait for that PR to be merged before merging here?

Reply:
Definitely wait. The upstream PR is merged now.